docs/plugins_guidelines.md

```diff
@@ -0,0 +1,81 @@
+# Third Party Plugin Guidelines
+
+Bevy has a plug and play architecture, where you can easily add plugins for new features, or replace built-in plugins with your own.
+
+This document targets plugin authors.
+
+## Naming
+
+You are free to use a `bevy_xxx` name for your plugin, with the caveat "please be reasonable". If you are about to claim a generic name like `bevy_animation`, `bevy_color`, or `bevy_editor`... please ask first. The rational is explained [here](https://github.com/bevyengine/bevy/discussions/1202#discussioncomment-258907).
+
+## Promotion
+
+You can promote your plugin to Bevy's [communities](https://github.com/bevyengine/bevy#community):
+
+* Add it to [Awesome Bevy](https://github.com/bevyengine/awesome-bevy)
+* Announce it on [Discord](https://discord.gg/gMUk5Ph), in channel `#showcase`
+* Announce it on [Reddit](https://reddit.com/r/bevy)
+
+## Bevy Version Supported
+
+Indicating which version of your plugin works with which version of Bevy can be a great help for your users. Some of your user may be using an older version of Bevy for any number of reason, you can help them finding which version of your plugin they should use. This can be shown as a simple table in your readme with each version of Bevy with a working version of your plugin.
```

Comment:
(grammar, readability)
Indicating which version of your plugin works with which version of Bevy can be **_helpful_** for your users. Some of your **_users_** may be using an older version of Bevy for any number of **_reasons_**, you can help them **_find_** which version of your plugin they should use. This can be shown as a simple table in your readme with each version of Bevy **_and the corresponding compatible_** version of your plugin.